proposals/vocabularies-adr.md

```diff
@@ -0,0 +1,78 @@
+# [short title of solved problem and solution]
+
+* Status: proposed
+* Deciders: @gregsdennis, @jdesrosiers
+* Date: 2024-06-10
+
+Technical Story:
+
+- Issues discussing feature - https://github.com/json-schema-org/json-schema-spec/issues?q=is%3Aopen+is%3Aissue+label%3Avocabulary
+- ADR to extract from the spec and use feature life cycle - https://github.com/json-schema-org/json-schema-spec/pull/1510
+
+## Context and Problem Statement
+
+The current approach to extending JSON Schema by providing custom keywords is
+very implementation-specific and therefore not interoperable.
+
+To address this deficiency, this document proposes vocabularies as a concept
+and a new Core keyword, `$vocabulary` to support it.
+
+## Decision Drivers <!-- optional -->
+
+- Language-agnostic
+- Ease of use
+- Ease of implementation
+
+## Considered Options
+
+### Current design as included in 2019-09 and 2020-12.
+
+Vocabularies are collections of keywords and are defined by vocabulary document.
```

Comment:
```suggestion
Vocabularies are collections of keywords and are defined by a vocabulary document.
```

Reply:
We need to keep the plurality consistent.  Do you have a preference between "vocabularies are..." and "a vocabulary is..."?

Reply:
Apparently I read carefully enough to notice that there was an mistake, but not carefully enough to make the right suggestion :man_facepalming:.

No strong preference from me, but I'd probably choose the singular.

Reply:
Updated to singular form